interpreter/cling/include/cling/Interpreter/IncrementalCUDADeviceCompiler.h

```diff
@@ -62,16 +62,15 @@ namespace cling {
                        const uint32_t fatbinFlags, const bool verbose,
                        const bool debug,
                        const std::vector<std::string> additionalPtxOpt)
-          : cppStdVersion(cppStdVersion),
-            hostTriple(hostTriple), smVersion(smVersion),
-            fatbinFlags(fatbinFlags), verbose(verbose), debug(debug),
-            additionalPtxOpt(additionalPtxOpt) {}
+          : cppStdVersion(cppStdVersion), hostTriple(hostTriple),
+            smVersion(smVersion), fatbinFlags(fatbinFlags), verbose(verbose),
+            debug(debug), additionalPtxOpt(additionalPtxOpt) {}
```

Comment:
This formatting change is unrelated, as are many other modifications in this file. Please revert from this PR, and configure your editor / IDE to only (re)format lines that are modified.